packages/views.client.unstable/src/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export {
+  type ArrayParameterValue,
+  type AsyncFailedValue,
+  type AsyncLoadedValue,
+  type AsyncLoadingValue,
+  type AsyncNotStartedLoadingValue,
+  type AsyncParameterValueMap,
+  type AsyncReloadingValue,
+  type AsyncValue,
+  type BooleanParameterValue,
+  type DateParameterValue,
+  type HostMessage,
+  type HostParametersUpdatedMessage,
+  isHostParametersUpdatedMessage,
+  isViewEmitEventMessage,
+  isViewReadyMessage,
```

Comment:
Do we need to expose the type guards for view-sourced events on the client library? An event constructor would be useful

Reply:
yea i can add more